### PR TITLE
Used axiosRetry to have a stable Commerce Mock Loki test

### PR DESCRIPTION
**Pull request by jeremyharisch**
**Description**
Regarding #11941 the check `Logs from commerce mock pod should be retrieved through Loki` from the commerce mock test fails sometimes. The failing pipelines do not give us any information why they failed. This PR changes the test to make use of the `axios` library for retries. Thus we get more information and a more resilient/stable test in general.

Changes proposed in this pull request:

- Usage of `axiosRetry`
- Log erros

**Related issue(s)**

#11941

### CODE REVIEW

File: tests/fast-integration/logging/client.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
-const axios = require('axios');
 
 const {
-    kubectlPortForward,
+    kubectlPortForward, getResponse,
```
Comment (shorim) [inline tests/fast-integration/logging/client.js:3]: just for having consistent style, can you please place `getResponse` in a new line?

File: tests/fast-integration/monitoring/client.js [hunk 2]
```diff
@@ -14,41 +14,34 @@ function prometheusPortForward() {
 }
 
 async function getPrometheusActiveTargets() {
-    let responseBody = await get("/api/v1/targets?state=active");
-    return responseBody.data.activeTargets;
+    let path = "/api/v1/targets?state=active"
```
Comment (shorim) [inline tests/fast-integration/monitoring/client.js:17]: There are missing semi-colons for the decalations of `path` and `url`. Same applies in the other functions.

**Review by shorim — APPROVED** [review]

**Review by skhalash — APPROVED** [review]

**Pull request merged**